src/lib/stack/client.ts

```diff
@@ -12,6 +12,7 @@ const REGION_URL_MAPPING: RegionUrlMap = {
   eu: 'eu-cdn.contentstack.com',
   'azure-na': 'azure-na-cdn.contentstack.com',
   'azure-eu': 'azure-eu-cdn.contentstack.com',
+  dev18: 'dev18-cdn.csnonprod.com',
```

Comment:
do we need to add dev host here

Reply:
@shafeeqd959 Yes, taxonomy isn't released yet. Due to the delivery SDK's use of the static host name ([csnonprod.com](http://csnonprod.com/)). Will remove this once taxonomy released.